Second_Seminar/app/src/main/java/org/sopt/androidseminar/ui/HomeActivity.kt

```diff
@@ -0,0 +1,94 @@
+package org.sopt.androidseminar.ui
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.androidseminar.R
+import org.sopt.androidseminar.adapters.RepositoryAdapter
+import org.sopt.androidseminar.data.RepositoryInfo
+import org.sopt.androidseminar.databinding.ActivityHomeBinding
+
+class HomeActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityHomeBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityHomeBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        Log.d(ACTIVITY_NAME, "onCreate")
+
+        setRepoRv()
+
+        moreButtonClickedEvent()
```

Comment:
불필요한 Spacing은 굳이 안해도 될 것 같아
```suggestion
        setRepoRv()
        moreButtonClickedEvent()
```